src/UI/Buyer/src/app/components/checkout/checkout/checkout.component.ts

```diff
@@ -99,7 +100,8 @@ export class OCMCheckout implements OnInit {
     private toastrService: ToastrService,
     private router: Router,
     private translate: TranslateService,
-    private mootrack: MooTrackService
+    private mootrack: SitecoreSendTrackingService,
```

Comment:
Should we change the private variable name as well?

Reply:
good call, resolved